staticServer/router.go

```diff
@@ -1,99 +1,66 @@
 package staticServer
 
 import (
-	"encoding/json"
-	"fmt"
 	"net/http"
+	"os"
 	"sync"
 
-	config "github.com/ASeegull/SmartFridge/staticServer/staticServerConfig"
+	config "github.com/ASeegull/SmartFridge/staticServer/config"
 	"github.com/davecheney/errors"
 	"github.com/gorilla/mux"
 	log "github.com/sirupsen/logrus"
 )
 
-func newRouter() *mux.Router {
-	cfg := config.GetStaticFilesPath()
+// Saves http methods to constants with short names
+const (
+	GET = http.MethodGet
+)
+
+// newRouter сreates and returns gorilla router
+func newRouter(staticPath string) *mux.Router {
 	r := mux.NewRouter()
-	r.Path("/").Handler(http.FileServer(http.Dir(cfg.Path)))
-	r.PathPrefix(cfg.Prefix).Handler(http.StripPrefix(cfg.Prefix, http.FileServer(http.Dir(cfg.Path))))
-	r.Path("/main").HandlerFunc(ServeInternal).Methods("GET")
-	r.Path("/login").HandlerFunc(LoginHandler).Methods("POST")
-	r.Path("/signup").HandlerFunc(SignUpHandler).Methods("POST")
-	r.Path("/content").HandlerFunc(Fetch("/client/fridgeContent")).Methods("GET")
-	r.Path("/recipes").HandlerFunc(Fetch("/client/allRecipes")).Methods("GET")
+	staticDir := http.FileServer(http.Dir(staticPath))
+	r.Handle("/", staticDir)
+	r.PathPrefix("/{_:.*}").Handler(staticDir)
 	return r
 }
 
 // Run starts servers for http and https connections
-func Run() {
+func Run(cfg *config.Config) {
+
 	var wg sync.WaitGroup
+
 	wg.Add(2)
 
-	var err error
-	r := newRouter()
+	r := newRouter(cfg.StaticPath)
+	port := os.Getenv("PORT")
+	switch {
+	case port != "":
+		log.WithField("port", port).Info("Server started")
+		log.Fatal(http.ListenAndServe(":"+port, r))
```

Comment:
If execution will be blocked on this ListenAndServe, why do we need next two goroutines?